scan/src/components/modal/ValidatorStakingInfo.re

```diff
@@ -199,18 +209,33 @@ module StakingInfo = {
                | _ => <DisplayBalance.Loading />
                }}
             </div>
-            <Button
-              px=20
-              py=5
-              onClick={_ => withdrawReward()}
-              disabled={
-                switch (allSub) {
-                | Data((_, balanceAtStake, _)) => balanceAtStake.reward.amount <= 0.
-                | _ => true
-                }
-              }>
-              <Text value="Withdraw Reward" weight=Text.Medium nowrap=true block=true />
-            </Button>
+            <div className={CssHelper.flexBox()} id="withdrawRewardContainer">
+              <Button
+                px=20
+                py=5
+                onClick={_ => withdrawReward()}
+                disabled={
+                  switch (allSub) {
+                  | Data((_, balanceAtStake, _)) => balanceAtStake.reward.amount <= 0.
+                  | _ => true
+                  }
+                }>
+                <Text value="Withdraw Reward" weight=Text.Medium nowrap=true block=true />
+              </Button>
+              <HSpacing size=Spacing.sm />
+              <Button
+                px=20
+                py=5
+                onClick={_ => reinvest()}
+                disabled={
+                  switch (allSub) {
+                  | Data((_, balanceAtStake, _)) => balanceAtStake.reward.amount <= 0.
+                  | _ => true
+                  }
+                }>
+                <Text value="Reinvest" weight=Text.Medium nowrap=true block=true />
+              </Button>
```

Comment:
1. Let's move switch on allSub above the `<div>` like this.
2. Pass the amount to reinvest function.
```
{switch(allSub) {
     | Data((_, balanceAtStake, _)) => {
          let disable = balanceAtStake.reward.amount <= 0.;
          <div className={CssHelper.flexBox()} id="withdrawRewardContainer">
               // render 2 button here and use disable variable.
          </div>
     }
     | _ => 
           <div className={CssHelper.flexBox()}>
               // render 2 button here and set disable as true
          </div>
     }
}}
```